src/auto-import.test.js

```diff
@@ -21,31 +21,37 @@ const { addImport, getImportDefinition } = require('./auto-import');
 
 // Dummy rule that executes the autoImport logic
 // for `dummy` and `extraDummy` variables.
-const dummyRule = ({ report, options }) => ({
+const dummyRule = {
```

Comment:
ESLint now errors and enforces the object form of rules rather than the function form.